### ISSUE TITLE
Why does the same data processing method produce different results (no denoising effect)?

### ISSUE THREAD

**Issue by uzizzz111** [sypsyp97/Filter2Noise#2]
I implemented a denoising pipeline and applied the same method  to two different datasets.
However, I found that the model works well on dataset A, but produces no noticeable denoising effect on dataset B.
The steps I fellowed include:
1. Get normalized .npy data from .IMA data
2. using the same loss function and training epochs
3. verifying that the model architecture is unchanged

The dataset I used is the AAPM 2016 Low-dose CT Grand Challenge.
Here are some results.

![Image](https://github.com/user-attachments/assets/00b3f7cf-705c-4fdd-902c-af51bf11330d)

![Image](https://github.com/user-attachments/assets/ae2e729a-d9e4-4474-8011-bd4994c751ab)

**Comment by sypsyp97** [comment]
Hi!

Thanks for your interest. You can try to set a smaller lambda.

Hope that will help 😀 

Best,
Yipeng
